hypervisor/arch/x86/guest/virq.c

```diff
@@ -508,9 +510,12 @@ static void vcpu_complete_splitlock_emulation(struct acrn_vcpu *cur_vcpu)
 	if (cur_vcpu->vm->hw.created_vcpus > 1U) {
 		foreach_vcpu(i, cur_vcpu->vm, other) {
 			if (other != cur_vcpu) {
+				bitmap_clear_lock(ACRN_REQUEST_SPLIT_LOCK, &other->arch.pending_req);
```

Comment:
also need to clear it here. otherwise, may cause dead lock in some cases which may cause all vcpus waiting in root mode. We confirmed this from the test

Reply:
This is sufficient. We need a clear root cause explaination,

Reply:
comments added